internal/analytics/client/sync/reporters/pixel.go

```diff
@@ -7,32 +7,49 @@ import (
 	"os"
 
 	"github.com/ActiveState/cli/internal/analytics/dimensions"
+	"github.com/ActiveState/cli/internal/config"
 	"github.com/ActiveState/cli/internal/constants"
 	"github.com/ActiveState/cli/internal/errs"
+
+	configMediator "github.com/ActiveState/cli/internal/mediators/config"
 )
 
 type PixelReporter struct {
-	url string
+	cfg *config.Instance
```

Comment:
I think we can still keep the url here, no? It's static, it only changes when the config listener fires, at which point we can update the value here. The config listener is set at construction time, so we can just loose the config after the listener is placed.